cypress/pageObject/Patients/PatientEncounter.ts

```diff
@@ -12,7 +12,56 @@ export class PatientEncounter {
       .click();
     return this;
   }
+  clickMedicinesTab() {
+    cy.verifyAndClickElement('[data-cy="tab-medicines"]', "Medicines");
+    return this;
+  }
+  clickEditPrescription() {
+    cy.verifyAndClickElement('[data-cy="edit-prescription"]', "Edit");
+    return this;
+  }
+  addMedication(
+    medicineName,
+    dosage,
+    frequency,
+    instructions,
+    route,
+    site,
+    method,
+    notes,
+  ) {
+    cy.get('[data-cy="question-medication-request"]').click();
+    cy.get('[role="listbox"]')
```

Comment:
```suggestion
    cy.get('[role="listbox"]')
```

check already existing command function to interact with dropdown, replace every usage with command function